src/audio/copier/copier_dai.c

```diff
@@ -31,18 +31,18 @@ static uint32_t bitmask_to_nibble_channel_map(uint8_t bitmask)
 	return nibble_map;
 }
 
-static int copier_set_alh_multi_gtw_channel_map(struct comp_dev *parent_dev,
```

Comment:
a part of this is in the previous commit. Can you please reorganize to move all of the same change to the same patch?

Reply:
done